committeeoversightapp/management/commands/import_data.py

```diff
@@ -0,0 +1,213 @@
+        return IndexError
+    else:
+        return ValueError
+
+def new_committee(committee_key, event, row_count):
+    try:
+        name = Committee.objects.get(lugar_id=committee_key).organization.name
+        organization = Committee.objects.get(lugar_id=committee_key).organization
+        entity_type = "organization"
+        committee, created = EventParticipant.objects.get_or_create(name=name, event=event, organization=organization, entity_type=entity_type)
+
+    except (ObjectDoesNotExist, AttributeError, ValueError) as e:
+        bad_rows.append("Row " + str(row_count) + ": Bad committee key " + committee_key)
+
+class Command(BaseCommand):
+    help = "Import Lugar spreadsheets data"
+
+    def handle(self, *args, **options):
+
+        # Create commmittees from key
+        self.stdout.write(str(datetime.now()) + ': Creating House committees from key...')
+        bad_rows.append("House Committees\n")
+        self.add_house_committees()
+        self.stdout.write(self.style.SUCCESS(str(datetime.now()) + ': House committees imported successfully!'))
+
+        self.stdout.write(str(datetime.now()) + ': Creating Senate committees from key...')
+        bad_rows.append("\nSenate Committees\n")
+        self.add_senate_committees()
+        self.stdout.write(self.style.SUCCESS(str(datetime.now()) + ': Senate committees imported successfully!'))
+
+        # Create hearings
+        self.stdout.write(str(datetime.now()) + ': Creating database entries for the House...')
+        bad_rows.append("\nHouse Hearings\n")
+        self.add_house_hearings()
+        self.stdout.write(self.style.SUCCESS(str(datetime.now()) + ': House hearings imported successfully!'))
+
+
+        self.stdout.write(str(datetime.now()) + ': Creating database entries for the Senate...')
+        bad_rows.append("\nSenate Hearings\n")
+        self.add_senate_hearings()
+        self.stdout.write(self.style.SUCCESS(str(datetime.now()) + ': Senate hearings imported successfully!'))
+
+        # Write bad category rows to file
+        with open('bad_rows.txt', 'a') as f:
+            for row in bad_rows:
+                f.write(row + '\n')
+
+    def add_house_committees(self):
+        house = Organization.objects.get(name="United States House of Representatives")
+
+        with open('data/final/house_committees_edited.csv', 'r') as csvfile:
+            reader = csv.DictReader(csvfile)
+
+            for row in reader:
+                committee_key = row['Committees and Subcommittees']
+                committee_name = row['b']
+                classification = "committee"
+
+                if committee_key:
+                    try:
+                        # ignore subcommittees that are actually a full committee meeting
+                        if committee_name == "Full Committee" or committee_name == "Full Commission":
+                            pass
+                        else:
+                            # full committees are three digits
```

Comment:
I'd like to see a slightly more verbose comment here, explaining where those "three digits" come from and perhaps citing an example. Otherwise, you're just summarizing what the code already says.